scrubble-client/src/html/game.html

```diff
@@ -1,5 +1,61 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Scrubble</title>
+  </head>
   <body>
-    <div id="pixi-application"></div>
+    <header class="header">
+      <p class="header__logo">Scrubble</p>
+    </header>
+    <section class="room-code">
+      <h2 class="room-code__label">Room Code:</h2>
+      <h2 class="room-code__code">WXYZ</h2>
+    </section>
+    <section class="players">
+      <div class="players__current-turn" id="players__1">
```

Comment:
I'd probably make sure that this also has the `players__player` class for consistency. In the BEM structure I think `current-turn` would probably be best suited as a modifier, e.g. `players__player players__player--current-turn`